distsql/select_result.go

```diff
@@ -140,7 +140,6 @@ type selectResult struct {
 	feedback     *statistics.QueryFeedback
 	partialCount int64 // number of partial results.
 	sqlType      string
-	encodeType   tipb.EncodeType
```

Comment:
There is already a `encodeType` in `this.selectResp.DAGRequest`
This field is duplicated...